network-noipv4-httpks.ks.in

```diff
@@ -25,7 +25,8 @@ shutdown
 
 if [[ "@KSTEST_OS_NAME@" == "rhel" ]]; then
     check_device_ifcfg_value ens4 BOOTPROTO none
-    check_device_ifcfg_value ens5 BOOTPROTO none
+    # when updating real ONBOOT value via NM api BOOTOPROTO is removed from the ifcfg
```

Comment:
This is hard to read, could you please rephrase it to something like:
``BOOTPROTO is removed from the ifcfg when ONBOOT value is updated via NM api`` ?